extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -344,3 +351,15 @@ function getBufferedPercentage(videoEl) {
   }
   return bufferedSum / videoEl.duration;
 }
+
+/**
+ * Whether the video is loaded (so it should not be downgraded)
+ * @param {!Element} videoEl
+ * @return {boolean}
+ */
+function isVideoLoaded(videoEl) {
+  if (videoEl.hasAttribute(VIDEO_BUFFERED_DEBUG_ATTR)) {
```

Comment:
The contant rename will make this test much easier to understand for someone without context